packages/jetpack-blocks/babel.config.js

```diff
@@ -0,0 +1,60 @@
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				targets: { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] },
```

Comment:
Watch for https://github.com/Automattic/wp-calypso/pull/31536 in case this needs rebasing.

Reply:
In fact, I think we can remove this to let Babel rely on default `browserslist` config provided by the consumer (Jetpack).